recipes/semver200/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, patch
+from conans.tools import check_min_cppstd
+from conans import CMake
+
+required_conan_version = ">=1.47.0"
+
+class SemVer200Conan(ConanFile):
+    name = "semver200"
+    version = "1.1.0"
```

Comment:
```suggestion
```
No version declared, so we can re-use the same recipe file to any future version.